web/fido_app/models.py

```diff
@@ -138,3 +138,19 @@ class Interaction(db.Model):
 
     def __repr__(self):
         return f'<Session token {self.session_token} triggered event {self.event} at {self.timestamp}> on page {self.page}'
+
+class LoginAttempts(db.Model):
+    '''
+    Model associating email and date with login attempt counts,
+    both successful and not.
+    '''
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    # Identifiers
+    email = db.Column(db.String(80), nullable=False)
```

Comment:
Do we want to distinguish between failed FIDO vs password logins?

Reply:
I know we can't accurately count all FIDO failures, but our model might want to account for this, at least. One option is just to add a `password_successes`, `password_failures`, `fido_successes`, and `fido_failures` fields